### PR TITLE
Change definition of static linking

### PR DESCRIPTION
**Pull request by dbabokin**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -370,14 +370,10 @@ else()
     set_target_properties(${PROJECT_NAME} PROPERTIES LINK_FLAGS "-z noexecstack -z relro -z now -Wl,--gc-sections")
 endif()
 
-if (ISPC_STATIC_STDCXX_LINK)
+if (ISPC_STATIC_STDCXX_LINK OR ISPC_STATIC_LINK)
     set_target_properties(${PROJECT_NAME} PROPERTIES LINK_FLAGS "-static-libgcc -static-libstdc++")
```
Comment (aneshlya) [inline CMakeLists.txt]: Here set_target_properties setting will rewrite LINK_FLAGS, not append to them. Consider using target_link_options instead.

File: .travis.yml [hunk 2]
```diff
@@ -64,6 +64,9 @@ env:
 before_install:
   - sudo apt-get update
   - sudo apt-get install -y libc6-dev-i386 g++-multilib lib32stdc++6
+  - wget https://cmake.org/files/v3.17/cmake-3.17.0-Linux-x86_64.sh && mkdir /opt/cmake && sh cmake-3.17.0-Linux-x86_64.sh --prefix=/opt/cmake --skip-license && sudo ln -s /opt/cmake/bin/cmake /usr/local/bin/cmake && rm cmake-3.17.0-Linux-x86_64.sh
```
Comment (aneshlya) [inline .travis.yml]: Please update in the same way Dockerfile for Ubuntu

Comment (aneshlya) [reply .travis.yml]: BTW why do we use cmake-3.17 here when minimum required version is 3013?

Comment (dbabokin) [reply .travis.yml]: It's already in Ubuntu Dockerfile, I've borrowed it from there. 3.17 is just because it's the latest.

**Pull request merged**